easybuild/easyconfigs/c/CRISPResso2/CRISPResso2-2.1.2-foss-2020b-Python-2.7.18.eb

```diff
@@ -0,0 +1,54 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+easyblock = 'PythonPackage'
+
+name = 'CRISPResso2'
+version = '2.1.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/pinellolab/CRISPResso2/'
+description = """CRISPResso2 is a software pipeline designed to enable rapid and
+intuitive interpretation of genome editing experiments."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+sources = [
+    {'source_urls': ['https://github.com/pinellolab/CRISPResso2/archive/'],
+     'filename': 'v%(version)s.tar.gz'},
+    {'source_urls': ['https://crispresso.pinellolab.partners.org/static/demo/'],
+     'filename': 'base_editor.fastq.gz', 'extract_cmd': 'cp %s %(builddir)s'}]
+patches = ['CRISPResso2-2.1.2-foss-2020b_remove_strict_deps.patch']
+checksums = [
+    '29dfaaa65f66225faee4887376ea6b524355332fc6f7ca4d07bb7d63596ef5e6',  # v2.1.2.tar.gz
+    '371aad07a164721188deaa80ca7589f5cd4b871119d90d55a984c6d9c8d31c7a',  # base_editor.fastq.gz
+    # CRISPResso2-2.1.2-foss-2020b_remove_strict_deps.patch
+    '87e4515d87c5e70fa65a8216a950dca5876948821ef5132a993102c168ece66a',
+]
+
+dependencies = [
+    ('Java', '11', '', True),
+    ('Python', '2.7.18'),
+    ('SciPy-bundle', '2020.11', '-Python-%(pyver)s'),
+    ('matplotlib', '2.2.5', '-Python-%(pyver)s'),
+    ('Trimmomatic', '0.39', '-Java-%(javashortver)s', True),
+    ('FLASH', '2.2.00'),
+    ('SAMtools', '1.12'),
+    ('Bowtie2', '2.4.2'),
+    ('Seaborn', '0.9.1', '-Python-%(pyver)s'),
+]
+download_dep_fail = True
+use_pip = True
+options = {'modulename': False}
```

Comment:
There's no sensible `import` check to do?

Should this be:

```python
options = {'modulename': name}
```

Reply:
nope, think it's not supposed to be used (by user) as imported module, though some importing happens on background. this is tested when --help is called though, so i guess no need to test it explicitly

Reply:
fixed in 37a8867fb8 using `options = {'modulename': name}`, doesn't hurt to do this extra check